env0/resource_environment_scheduling.go

```diff
@@ -69,6 +69,14 @@ func resourceEnvironmentSchedulingCreateOrUpdate(ctx context.Context, d *schema.
 		return diag.Errorf("schema resource data deserialization failed: %v", err)
 	}
 
+	if payload.Deploy != nil && !payload.Deploy.Enabled {
```

Comment:
This basically passes a null instead of a disabled cron.
Would have been better to fix it in the backend. However, for the sake of time I've fixed it here.